### PR TITLE
Unifying description of checkout procedure

### PR DESCRIPTION
**Pull request by gvwilson**
1.  Rename `11-checkout.md` to `checkout.md` (since it isn't really a lesson).
2.  Put all information about checkout procedure in `checkout.md`.
3.  Adjust standard messages to refer to this, rather than duplicating information.
4.  Including pointer in checklist to template for instructor training workshop websites (which needs updating).

@apawlik @ChristinaLK can you please review ASAP?  This synchronizes with https://github.com/swcarpentry/instructor-training-session-template/pull/2.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
+1 to moving 11-checkout.md to "non-lesson" file - it's not really lesson material.  

First pass looks good to me.  The only omission I see is people who want to certify for both DC/SWC and being clear that each step must be done for both if you want to certify for both.  I can suggest more helpful verbiage later...

Also, it feels very long and text-heavy.  I'll see if I can think of a way to shorten/condense while still being sufficiently detailed/thorough.

**Comment by apawlik** [comment]
Thanks for feedback. I'm incorporating it now.

File: 11-checkout.md [hunk 1]
```diff
@@ -1,119 +0,0 @@
...
-> ## Motivating Question
-> 
-> What are the final steps in becoming an instructor trainer?
-{: .question}
-
-The final steps in getting certified are as follows:
-
-1. Pick one of the lessons we are currently accepting trainee requests on
-   and submit a *small* change.
-   (Your trainer will provide you with a list of lessons.)
```
Comment (apawlik) [inline 11-checkout.md]: I'd rephrase: "Your trainer will provide you with a list of lessons which currently accept improvements and/or exercises. Pick one of them and submit a *small* change."

File: 11-checkout.md [hunk 2]
```diff
@@ -1,119 +0,0 @@
...
-   If you want to teach Software Carpentry,
-   you must submit the change as a pull request on GitHub
-   and send the link to your trainer.
```
Comment (apawlik) [inline 11-checkout.md]: Sorry to be a pain, I'd still say that we should have some general email such as training@... and trainees should cc this email in. If their trainer is not available (could be for a long while), they are stuck with their checkout.

File: 11-checkout.md [hunk 3]
```diff
@@ -1,119 +0,0 @@
...
-   If you want to teach Data Carpentry,
-   go to [its submission page][dc-submission-page]
-   (which will redirect you to the appropriate place).
-
-   Note:
```
Comment (apawlik) [inline 11-checkout.md]: Following the comment from @ChristinaLK on cutting down on text, I'd say that this note could possibly be removed or rather moved to the materials for the trainers. They need to know this and explain, if needs be.

File: 11-checkout.md [hunk 4]
```diff
@@ -1,119 +0,0 @@
...
-   Everybody will botch a commit message at some point.  This
-   exercise introduces them to an easy way to fix those mistakes.
-   ~~~
-
-   (as both a commit message and a pull-request summary) will be a lot
-   easier for the maintainer than:
-
-   ~~~
-   Instructor-training assignment
-   ~~~
```
Comment (apawlik) [inline 11-checkout.md]: Shouldn't this title be at the top? The submission to the materials is also an assignment. ?

File: 11-checkout.md [hunk 5]
```diff
@@ -1,119 +0,0 @@
...
-
-3. Pick one lesson from the core Software Carpentry lessons (if you
```
Comment (apawlik) [inline 11-checkout.md]: I think changing the header which says this is part two would be useful. People find this confusing (experience from live IT) - "But we already were told to pick one for submission. Is it the same?"

File: 11-checkout.md [hunk 6]
```diff
@@ -1,119 +0,0 @@
...
-   wish to teach Software Carpentry) and/or one from the core Data
-   Carpentry lessons (if you wish to teach Data Carpentry), and read
-   through it/them carefully.
-
-4. Sign up on the Etherpad indicated by your instructor to take part
-   in an hour-long discussion session about the lessons you are going
-   to use for your demo lesson (discussed below), and about the
```
Comment (apawlik) [inline 11-checkout.md]: I think 3 and 4 don't connect. It may be confusing. Maybe "to take part  in an hour-long discussion session about the lessons you chose in the previous step. You will also teach a short demo from these lessons (see below)."

File: 11-checkout.md [hunk 7]
```diff
@@ -1,119 +0,0 @@
...
-   Carpentries in general.
-
-   You are expected to be familiar with your chosen lesson(s) *before*
-   this discussion session.  If the discussion leader feels you are
-   not, you will encouraged to do more preparation and come back for a
-   later discussion session.  If you are still unprepared, you will be
-   asked to contact the header of instructor training to discuss
```
Comment (apawlik) [inline 11-checkout.md]: "the header"? "the head" or "the leader"? or sorry for my English.

File: 11-checkout.md [hunk 8]
```diff
@@ -1,119 +0,0 @@
...
-   wish to teach Software Carpentry) and/or one from the core Data
-   Carpentry lessons (if you wish to teach Data Carpentry), and read
-   through it/them carefully.
-
-4. Sign up on the Etherpad indicated by your instructor to take part
-   in an hour-long discussion session about the lessons you are going
-   to use for your demo lesson (discussed below), and about the
-   Carpentries in general.
-
-   You are expected to be familiar with your chosen lesson(s) *before*
```
Comment (apawlik) [inline 11-checkout.md]: Suggest: "You are expected to actively take part in the discussion session so you are expected to be familiar with your chosen lesson(s).

**Comment by apawlik** [comment]
Sorry about my missed comments. I do another pass with focus on cutting down on the text. I agree with @ChristinaLK that it's too long. Too much to read (I understand why we want to be detailed but still)

File: checkout.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+---
+After you have finished the two-day training class,
+you must go through these steps in order to complete your training and be certified:
+
+1.  Submit a [small change to one of our lessons](#lesson-change).
+2.  Take part in an online [discussion session](#discussion-session).
+3.  Teach a short [demonstration lesson](#demo-lesson) online.
+
+We would like to get everyone through this process within two months of training,
+so please get started as soon as you can.
```
Comment (apawlik) [inline checkout.md]: [reducing text] Delete: "so please get started as soon as you can."

File: checkout.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+Anyone who has not completed within 90 days should contact their trainer,
+or the head of instructor training,
+to discuss reasons and options.
+
+## Part 1: Submit a Small Change to One of Our Lessons
+<a name="lesson-change"></a>
+
+Our lessons are maintained and improved by the people who teach them,
+so this part of the checkout procedure gives you a chance to familiarize yourself
+with the way we manage that.
```
Comment (apawlik) [inline checkout.md]: suggest wording: "this part of the checkout procedure gives you a chance to familiarize yourself with our collaborative training material development process." (because we talk about *collaborative training material development* all the time; let's just use that terminology)

File: checkout.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+
+Pick one of the lessons from the list given later in this section
+and submit a *small* change.
+Note that you may submit a change to a Data Carpentry lesson
+when qualifying as a Software Carpentry instructor:
+the goal is to be sure you are familiar with the process.
+
+Your change can be a new exercise or diagram,
+or improve one or more of the lesson's learning objectives.
```
Comment (apawlik) [inline checkout.md]: "or an improvement of one..." (English?)

File: checkout.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+    you may submit the change either as a pull request
+    or by filing an issue in GitHub that describes your proposed change.
+    Either way,
+    send a link to the pull request or issue to [{{page.email}}]({{page.email}})
+    so that she can shepherd it through the review process.
+
+2.  Your change doesn't have to be accepted in order for you to become an instructor:
+    the point is to make sure that you understand the process and have met the people who review material.
+
+3.  Be sure to format your submission in the same way as the content that's already there.
```
Comment (apawlik) [inline checkout.md]: [reducing text] Delete: " For example, lessons should include the 'challenge' style, while learning objectives should be in the quoted block at the top of the lesson."
I know this is useful to have but hope that the next sentence will ensure people get it.

File: checkout.md [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+    (reversing a string isn't particularly motivational for our audience).
+
+8.  If you would rather do a diagram, we are happy to take submissions to any of our lessons.
+    Please use SVG, since vector formats are easier to manipulate than raster formats.
+
+9.  We also appreciate improvements to our lessons' learning objectives.
+    In particular, we would like checked against the actual lesson content,
+    and would also like them labelled according to the levels of Bloom's Taxonomy.
+
+The lessons you may submit to are:
```
Comment (apawlik) [inline checkout.md]: [Meta remark] Whilst this is useful, we need to make sure this list is always up to date.

File: checkout.md [hunk 14]
```diff
@@ -0,0 +1,225 @@
...
+   Which bits of the module may be tricky and if poorly explained, they may lead the students to broken mental model? 
+4. What prior knowledge do learners need to have before participating in this module (whole lesson)?
+   If you were to list any prerequsites, what would they be? 
+
+The first two questions are intended to help you become more comfortable with
+[our collaborative lesson development methods]({{site.site}}/blog/2014/03/collaborative-lesson-development.html).
+The third and fourth question refer back to the instructor training course,
+and are meant to give you a chance to apply some of what you've learned to these specific lessons.
+
+Once you are familiar with a lesson,
```
Comment (apawlik) [inline checkout.md]: Turn this into bullet points for easier reading? I think all these are relevant so even though we'd like to reduce the amount of text, I can't see anything to take out here :-(

File: checkout.md (hunk 9 above)
Comment (gvwilson) [inline checkout.md]: I'm not so worried about shortening this - trainees will be reading it on their own time, and if you think of it as a FAQ meant to anticipate and answer their questions, it really does have to spell things out in detail.  (Most of the points are taken directly from questions we've had and answers we've sent...)

File: checkout.md (hunk 11 above)
Comment (gvwilson) [inline checkout.md]: The grammar here is "Your change can (be a new...) or (improve one or more...)", but I agree, it'll read more easily as "Your change be X or Y", so I'll do that.

File: checkout.md (hunk 12 above)
Comment (gvwilson) [inline checkout.md]: This is the most common formatting mistake I've been seeing, so I wanted to highlight it specifically.

File: checkout.md (hunk 13 above)
Comment (gvwilson) [inline checkout.md]: Yup - that's why I want the SWC and DC sites to link to this doc, and the follow-up emails we send trainees to point to it, so that there's one authoritative source to update.

File: checkout.md (hunk 14 above)
Comment (gvwilson) [inline checkout.md]: Will do.

File: checkout.md (hunk 12 above)
Comment (apawlik) [inline checkout.md]: Fair enough!

File: checkout.md [hunk 15]
```diff
@@ -0,0 +1,250 @@
...
+> Note that we will pick the segment of the lesson you are to teach
+> on the day of the demonstration,
+> so you must be prepared to teach any part of your chosen lesson.
+
+For your demonstration,
+you will screen-share through Google Hangouts
+and live code as if your computer was plugged into a projector
+and you were standing in front of a class.
+You may use whatever notes you want,
+provided they are not visible on your screen.
```
Comment (apawlik) [inline checkout.md]: I'm bit wary about this. Recently I did a checkout when one of the attendees used RStudio and had opened the lesson in the right hand panel (where you can preview websites). He wasn't reading from the notes and I found this solution really useful for normal teaching. Not just for the instructor but also for the learners, they often like to see the notes and  having them opened in the panel reduces the problem of constant swapping between windows. 

Having said that, I think this is mainly possible with RStudio (in our context).

Comment (gvwilson) [inline checkout.md]: I think that once someone has enlarged their font enough for the screen 
to be readable when projected, side-by-side is impossible (at least on a 
single-screen system - if they're lucky enough to have two projectors, 
that changes everything).

File: checkout.md [hunk 16]
```diff
@@ -0,0 +1,250 @@
...
+demopad: http://pad.software-carpentry.org/teaching-demos
+discussionpad: http://pad.software-carpentry.org/lesson-discussion-2016
+email: training-help@software-carpentry.org
+---
+After you have finished the two-day training class,
+you must go through these steps in order to complete your training and be certified:
+
+1.  Submit a [small change to one of our lessons](#lesson-change).
+2.  Take part in an online [discussion session](#discussion-session).
+3.  Teach a short [demonstration lesson](#demo-lesson) online.
```
Comment (apawlik) [inline checkout.md]: We need to be clear that they need to complete steps 2 and 3 for SWC and DC separately(?) if they want to be certified for both. At the moment this is not very clear and the details of (1) below are even more confusing ("Note that you may submit a change to a Data Carpentry lesson when qualifying as a Software Carpentry instructor").

File: checkout.md [hunk 17]
```diff
@@ -0,0 +1,250 @@
...
+The first two questions are intended to help you become more comfortable with
+[our collaborative lesson development methods]({{site.site}}/blog/2014/03/collaborative-lesson-development.html).
+The third and fourth question refer back to the instructor training course,
+and are meant to give you a chance to apply some of what you've learned to these specific lessons.
+
+Once you are familiar with a lesson:
+
+1. Add yourself to an upcoming discussion session on [this Etherpad]({{page.discussionpad}})
+   (or put yourself in the list at the top
+   if there isn't an upcoming discussion session for your lesson that fits your schedule).
```
Comment (apawlik) [inline checkout.md]: Should we say something about the limit of people per session? 5?

File: checkout.md [hunk 18]
```diff
@@ -0,0 +1,250 @@
...
+(Note that any particular trainee will only be asked some of these questions,
+but they should prepare answers to all of them.)
+
+## Part 3: Teach a Short Demonstration Lesson
+<a name="demo-lesson"></a>
+
+Once you have completed your discussion session,
+the final step is to sign up for a 5-minute online demonstration lesson
+[this Etherpad]({{page.demopad}})
+(or add yourself to the top of the pad if none of the available times work for you).
```
Comment (apawlik) [inline checkout.md]: Should we say something about the limit of people per session? 5?

File: checkout.md (hunk 15 above)
Comment (apawlik) [inline checkout.md]: OK!

File: instructors.md [hunk 19]
```diff
@@ -426,85 +427,70 @@ and have a rubric to follow.
...
-    collected in one document.
-
-*   The discussion lead is free to use the Software Carpentry persistent
-    BlueJeans connection, or other options such as Google Hangouts or
-    Skype if they prefer.  The lead must post information about the
-    connection to the discussion session's Etherpad so that people
-    will know how to join.
-
-### During:
+The discussion session is meant to give students
```
Comment (apawlik) [inline instructors.md]: students -> trainees ? (nit picking, I know)

File: instructors.md [hunk 20]
```diff
@@ -426,85 +427,70 @@ and have a rubric to follow.
...
-        selected by the trainer.
-        The trainee will *not* know in advance which 5 minutes she will be asked to teach
-        (which encourages her to prepare the entire lesson).
-
-    *   Everyone will be notified afterwards by email whether they
-        passed or not.
-
+*   Briefly review the [checkout procedure](checkout.html).
+    Make sure trainees understand that *they* pick the lesson,
+    but *we* pick the segment from that lesson.
```
Comment (apawlik) [inline instructors.md]: Does the "we pick the segment" really apply to the discussion session? It's about the demo only?

File: instructors.md [hunk 21]
```diff
@@ -426,85 +427,70 @@ and have a rubric to follow.
...
+Note:
 
-*   It is important that we ensure that all of the participants get to
-    discuss their issues.  In cases where all participants are vocal,
-    a structured walkthrough may not be necessary.
+1.  It is important that we ensure that *all* trainees get to raise issues.
+    If some trainees are more vocal than others,
+    or if trainees are mixed with experienced instructors (in debriefing sessions),
+    the discussion leader must moderate and ask leading questions
+    to give everyone a fair chance to speak.
```
Comment (apawlik) [inline instructors.md]: What I've tried is "round table". I pick an open-ended question from the rubric and asked the trainees one after another (I said we'd do "round table"). Then (depending on the group) the discussion session becomes more free but the leader still needs to act as the moderator. I think the key is to ask directly those who tend to be quiet.

File: instructors.md [hunk 22]
```diff
@@ -426,85 +427,70 @@ and have a rubric to follow.
...
+2.  We do not need to ask every question of every trainee,
+    just as we do not ask every trainee to teach an entire half-day lesson.
+    Try instead to mix up the questions to check that the trainees have prepared adequately.
```
Comment (apawlik) [inline instructors.md]: What I've tried is starting with a "round table". I pick an open-ended question from the rubric and asked the trainees one after another (I said we'd do "round table"). Then (depending on the group) the discussion session becomes more free but the leader still needs to act as the moderator. I think the key is to ask directly those who tend to be quiet.

File: checkout.md (hunk 16 above)
Comment (ChristinaLK) [inline checkout.md]: +1
I'd be okay with merging as is (just to have a more updated version on the website) and then finessing the language in another PR.

**Comment by ChristinaLK** [comment]
This all looks good to me, except for one comment, which I've noted above.  I still think we might need extra-clear instructions for the volunteers who lead the debriefing sessions (what they need to do re: trainees), but that's something for me to coordinate with the mentoring committee.
